sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -40,7 +40,7 @@ import (
 var ecrRegex = regexp.MustCompile(`(^[a-zA-Z0-9][a-zA-Z0-9-_]*)\.dkr\.ecr\.([a-zA-Z0-9][a-zA-Z0-9-_]*)\.amazonaws\.com(\.cn)?.*`)
 
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
I think we should just not seed correct? I don't think its needed anymore. Can you confirm either way with the docs for this lint?

Reply:
I just re-read the docs, you have it correct now.